### PR TITLE
ENH: Makefile

### LINKED ISSUE #2: Project should have a Makefile
Include all the commands that need to be executed by the system.

### PR DESCRIPTION
**Pull request by turanoo**
Fixes #2

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (thermokarst) [inline Makefile]: Can you remove this extra blank line?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	@echo "make dev    -------->> to install yarn package manager."
+
+lint:
+	npm run lint
+
+build:
+	npm run build
+
+deploy:
+	./node_modules/.bin/clasp login
```
Comment (thermokarst) [inline Makefile]: On second thought, let's split these two commands into two separate directives `deploy` & `login`, does that make sense?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	cd dist && ./../node_modules/.bin/clasp push && cd ..
+
+dev:
+	install yarn
```
Comment (thermokarst) [inline Makefile]: Should this be `yarn install`?

**Review by thermokarst — CHANGES_REQUESTED** [review]
Thanks @turan1393 - I have a few changes requested inline. As well, some (or all) of the directives in this Makefile will need a `PHONY` config (see [here](https://github.com/qiime2/vm-playbooks/blob/master/Makefile#L6) for an example). You can read up on Makefiles a bit [here](https://www.gnu.org/software/make/manual/make.html). Thanks!

**Comment by turanoo** [comment]
@thermokarst,

Thank you for comments. I will make those changes and push the file again. Thank you.

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,27 @@
+.PHONY: help
+help:
+	@echo "make build  -------->> to compile the code."
+	@echo "make lint   -------->> to check code quality."
+	@echo "make login  -------->> to log in to your google account."
+	@echo "make deploy -------->> to push the code to appscript."
+	@echo "make dev    -------->> to install yarn package manager."
```
Comment (thermokarst) [inline Makefile]: how about "to install dependencies using yarn."?

**Review by thermokarst — CHANGES_REQUESTED** [review]
The `Makefile` looks great and works well! Can you please update the README to use the `make blah` directives where applicable?

File: Makefile (hunk 4 above)
Comment (turanoo) [reply Makefile]: Yes, I will commit the new changes in a few. 💯

**Review by thermokarst — APPROVED** [review]

**Pull request merged**